docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -24,52 +26,58 @@ In BODS, the following properties are used to represent such information disclos
 
 * ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
 * ``directOrIndirect`` (See :ref:`schema-interest`)
-* ``componentStatementIDs`` (See :ref:`schema-relationship-record`)
+* ``componentRecords`` (See :ref:`schema-relationship-record`)
 * ``isComponent`` (See :ref:`schema-entity-record`)
 
 Requirements
 ------------------------
 
-The beneficial ownership relationship
+No beneficial ownership relationships
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If an entity declares that it has no disclosable beneficial owners, its record MUST be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'noBeneficialOwners'.
+
+If an entity is exempt from declaring its beneficial owners its record SHOULD be the ``subject`` of a Relationship statement with ``recordDetails.interestedParty.reason`` 'subjectExemptFromDisclosure'.
+
+
+A beneficial ownership relationship
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If a person is a beneficial owner of an entity, entity X, whether directly or indirectly, and one of them is required to declare this beneficial ownership:
```

Comment:
```suggestion
If a person is a beneficial owner of an entity (directly or indirectly) and their relationship is declared there MUST be a *primary* relationship connecting the two which represents the beneficial ownership relationship.
```
I don't think entity X is useful here. The numbering doesn't make sense without point 2 so might be neater to have it all in one sentence. 

should this be Relationship Statement / Relationship Record? Just to clearly distinguish from the irl relationships?

Reply:
Yes, the 'entity X' thing is unpleasant. I introduced it for clarity because we have to refer back to this entity from the Intermediaries section. If there's a better way to communicate this, let edit.

I've taken your point about the numbering and merged the first two clauses.

Reply:
Yeah that's fair - I've tried to think of a better way and I can't think of anything significantly better atm